STROOP/STROOP.csproj

```diff
@@ -10,6 +10,7 @@
     <RootNamespace>STROOP</RootNamespace>
     <AssemblyName>STROOP</AssemblyName>
     <TargetFrameworkVersion>v4.8</TargetFrameworkVersion>
+    <LangVersion>11</LangVersion>
```

Comment:
This breaks support for Visual Studio 2017. While I understand the want to use some of the more concise syntax that newer C# versions bring, that change should be part of a separate issue that would also specify and apply related coding styles (e.g. nullability annotations for reference types, pattern matching vs. equality checks, etc.).